tests/models/test_image.py

```diff
@@ -19,6 +19,13 @@
 from models.image import Image
 from models.instrument import Instrument, get_instrument_instance
 
+# Have to have this here; otherwise, decam.py never gets loaded, and
+# DECam never gets added to the global instrument.INSTRUMENT_INSTANCE_CACHE
+#
+# There must be a better solution.  Do we just need to stuff all of the instruments
+# in the same file?  Or, should we rerun register_all_instruments() at the bottom
+# of every instrument's .py file?
+import models.decam
```

Comment:
I also ran into this. I think this only comes up in tests where not every model gets initialized when the app comes up. 
I think the solution (which I believe I added to my MD5sum PR) is to simply import these instruments in conftest.py.